controllers/Category.js

```diff
@@ -53,3 +58,68 @@ exports.showAllCategories = async (req, res) => {
     });
   }
 };
+
+//categoryPageDetails Handler function
+exports.categoryPageDetails = async (req, res) => {
+  try {
+    //fetch categoryId from req.body
+    const { categoryId } = req.body;
+
+    // Get courses for the specified category
+    const selectedCategory = await Category.findById(categoryId)
+      .populate("courses")
+      .exec();
+    console.log(selectedCategory);
```

Comment:
dev logs

Reply:
the console logs are more seen like developer added them for testing, and by mistake it has been committed

Reply:
should i remove them, i think i need to keep them till I go through testing api phase, What do you think?